pyccel/codegen/printing/fcode.py

```diff
@@ -53,7 +52,7 @@
 from pyccel.ast.datatypes import NativeRange, NativeTuple
 from pyccel.ast.datatypes import CustomDataType
 
-from pyccel.ast.internals import Slice
+from pyccel.ast.internals import Slice, PyccelSymbol
```

Comment:
Codacy was rasing an `unused-import` error for this. I'm not sure where it's gone but it still seems to be true

Reply:
its not gone

Reply:
It was gone in commit 71d44cf but it's back ^^